public/js/actions/navigation.js

```diff
@@ -19,7 +19,19 @@ function navigate() {
   };
 }
 
+function reload() {
```

Comment:
I'm wary of adding actions that aren't explicitly for users to use. This is just for testing, so I'd rather take a different approach.  Let's just expose the client to the mochitests just like we expose the other things, and then tests can just call `client.reload()` themselves. What do you think?